cli/tests/coverage_tests.rs

```diff
@@ -26,6 +26,62 @@ mod coverage {
     no_snaps_included("no_snaps_included", "ts");
   }
 
+  #[test]
+  fn error_if_invalid_cache() {
+    let deno_dir = TempDir::new();
+    let tempdir = TempDir::new();
+    let tempdir = tempdir.path().join("cov");
+
+    let invalid_cache_path =
+      util::testdata_path().join("coverage/invalid_cache");
+    let mod_before_path = util::testdata_path()
+      .join(&invalid_cache_path)
+      .join("mod_before.ts");
+    let mod_after_path = util::testdata_path()
+      .join(&invalid_cache_path)
+      .join("mod_after.ts");
+    let mod_path = util::testdata_path()
+      .join(&invalid_cache_path)
+      .join("mod.ts");
+
+    // Write the inital mod.ts file
+    std::fs::copy(mod_before_path, &mod_path).unwrap();
```

Comment:
Could we copy this to the temp directory instead? That will also allow us to remove mod.ts from git.